src/pages/Detail/Header.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+
+const Header = headerData => {
+  const { title, name, hash } = headerData;
+  return (
+    <>
+      <Title>{title}</Title>
+      <SubTitle>{name}</SubTitle>
+      <HashTagSection>
+        {hash.map((item, i) => {
+          const { text } = item;
+          return <HashTag key={i}>{text}</HashTag>;
+        })}
```

Comment:
매개변수자리에서 바로 구조 분해 할당을 해주셔도 됩니다!
```suggestion
        {hash.map(({text}, i) => {
          return <HashTag key={i}>{text}</HashTag>;
        })}
```